ui/src/components/ActiveAlertsFilter.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Dropdown } from '@scality/core-ui';
+import { useHistory } from 'react-router';
+import { useQuery } from '../services/utils';
+import { padding } from '@scality/core-ui/dist/style/theme';
+import { STATUS_WARNING, STATUS_CRITICAL } from '../constants.js';
+
+export const FilterIcon = styled.i`
+  color: ${(props) => {
+    const theme = props.theme.brand;
+    let color = theme.textPrimary;
+
+    switch (props.status) {
+      case STATUS_WARNING:
+        color = theme.warning;
+        break;
+      case STATUS_CRITICAL:
+        color = theme.danger;
+        break;
+      default:
+        color = theme.textPrimary;
+    }
+    return color;
+  }};
+  padding: ${padding.smaller};
+`;
+
+export const ActiveAlertsFilterWrapper = styled.div`
+  .sc-dropdown {
+    width: 140px;
+  }
+  .sc-dropdown > div {
+    background-color: ${(props) => props.theme.brand.info};
+    width: 140px;
+  }
+`;
+
+const ActiveAlertsFilter = (props) => {
+  const { baseLink } = props;
```

Comment:
I think it makes sense to remove the `baseLink` since we change in this component is Query string.